Updated Events page formatting and added Past Events section

tbg510 opened the pull request:
## Updated Events page formatting and added Past Events section
## Issue Number
Closes #149 
## Description
Updated Events page formatting to include some color and adjust the positions of event details. Added Past Events category in `EventsList` that lists any events with dates before today's date. Also updated an event's date in the JSON to be today's date so that there are events in each category (for today). 
## Testing
After this PR the Events page is pretty much finished besides any minor edits, so the tests for this page can finally be made (if anyone still wants to make them).
## Possible Errors
Shouldn't be any errors with the Past Events feature from what I've seen.
## Images
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/92517582/ee97ce9d-50b2-419a-94eb-5dad6181776d)
Updated Events page with formatting changes and new Past Events category that is unexpanded.
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/92517582/c9eea6fb-34cd-4a8a-a906-657b7c62d80c)
Events page with Past Events category expanded.

aiharc1 reviewed (approved):
Nice work Matthew. This is a good improvement to the previous version of this page. It is nicer on the eyes and easier to read.  Not sure if its just me but the orange is a little bright compared to the other pages. My suggestion would be to change it to one of the blues we use. Other than that lookin good.

tbg510 commented:
> Nice work Matthew. This is a good improvement to the previous version of this page. It is nicer on the eyes and easier to read. Not sure if its just me but the orange is a little bright compared to the other pages. My suggestion would be to change it to one of the blues we use. Other than that lookin good.

Yea I can change it to a different color, I just used the orange we used for the events icon/tab since I wasn't sure what color to use. I'll see if anyone else has feedback on the color.

junxjr reviewed (approved):
This looks good, each section is well organized by having its own box around it. I agree with cade, it is a little bright. I think maybe try light orange? or light blue.

Redblaze74 reviewed (approved):
LGTM, great work implementing the display, minor design things I'd like to note:
- Implementing the title component using `<Title page="Events">` and importing the component to use it
- Making the headers slightly bigger just so it isn't the same font size as the events/page content 
- Less saturated color for the events background, maybe you could implement a gradient using `bg-gradient-to-r from-(some less saturated color) to-(a more saturated color)`? the data.js file under /components has a list of gradients we are currently using for both titles and the header tabs

tbg510 commented:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/92517582/f42cea30-d869-47cb-a0cb-c48b242ddd87)
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/92517582/3bd20dc3-a7c3-4228-8d25-f022fb3ba37e)
I updated the formatting based on your guys' feedback. I added the title component at the top of the page. I increased the header font sizes to be bigger than the event details font. I changed the event background color to be a gradient instead of a solid color using `bg-gradient-to-r from-amber-400 to-orange-400` instead of the 500 used for the title. Let me know if there should be any more adjustments!

AlvaroH2001 reviewed (approved):
Great job on the changes to the Events page, it looks great and full of life and order. Good job on responding to feedback from the other members in a timely and responsible manner, we should all follow your example.
